instrumentation/opentelemetry-instrumentation-system-metrics/src/opentelemetry/instrumentation/system_metrics/__init__.py

```diff
@@ -81,155 +82,598 @@ def __init__(
+        # self._system_swap_page_operations = {}
+
+        self._system_disk_io_labels = {}
+        self._system_disk_operations_labels = {}
+        self._system_disk_time_labels = {}
+        self._system_disk_merged_labels = {}
+
+        # self._system_filesystem_usage_labels = {}
+        # self._system_filesystem_utilization_labels = {}
+
+        self._system_network_dropped_packets_labels = {}
+        self._system_network_packets_labels = {}
+        self._system_network_errors_labels = {}
+        self._system_network_io_labels = {}
+        self._system_network_connections_labels = {}
+
+        self._runtime_cpython_memory_labels = {}
+        self._runtime_cpython_cpu_time_labels = {}
+        self._runtime_cpython_gc_count_labels = {}
+
         # create the label set for each observer once
         for key, value in self._labels.items():
-            self._system_memory_labels[key] = value
-            self._system_cpu_labels[key] = value
-            self._network_bytes_labels[key] = value
-            self._runtime_memory_labels[key] = value
-            self._runtime_gc_labels[key] = value
+            self._system_cpu_time_labels[key] = value
+            self._system_cpu_utilization_labels[key] = value
+
+            self._system_memory_usage_labels[key] = value
+            self._system_memory_utilization_labels[key] = value
+
+            self._system_swap_usage_labels[key] = value
+            self._system_swap_utilization_labels[key] = value
+            # self._system_swap_page_faults[key] = value
+            # self._system_swap_page_operations [key] = value
+
+            self._system_disk_io_labels[key] = value
+            self._system_disk_operations_labels[key] = value
+            self._system_disk_time_labels[key] = value
+            self._system_disk_merged_labels[key] = value
+
+            # self._system_filesystem_usage_labels[key] = value
+            # self._system_filesystem_utilization_labels[key] = value
+
+            self._system_network_dropped_packets_labels[key] = value
+            self._system_network_packets_labels[key] = value
+            self._system_network_errors_labels[key] = value
+            self._system_network_io_labels[key] = value
+            self._system_network_connections_labels[key] = value
 
         self.meter.register_observer(
-            callback=self._get_system_memory,
-            name="system.mem",
-            description="System memory",
+            callback=self._get_system_cpu_time,
+            name="system.cpu.time",
+            description="System CPU time",
+            unit="seconds",
+            value_type=float,
+            observer_type=SumObserver,
+        )
+
+        self.meter.register_observer(
+            callback=self._get_system_cpu_utilization,
+            name="system.cpu.utilization",
+            description="System CPU utilization",
+            unit="1",
+            value_type=float,
+            observer_type=UpDownSumObserver,
```

Comment:
My bad @ocelotl, in OTEP 119 I think all of the `UpDownSumObserver` should be `ValueObserver` (see https://github.com/open-telemetry/opentelemetry-specification/issues/819).

The [metrics API](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/metrics/api.md#valueobserver) points out the similarity/confusion between `UpDownSumObserver` and  `ValueObserver` with the example of measuring queue size:
> Therefore, considering the choice between ValueObserver and UpDownSumObserver, the recommendation is to choose the instrument with the more-appropriate default aggregation. If you are observing a queue size across a group of machines and the only thing you want to know is the aggregate queue size, use SumObserver because it produces a sum, not a distribution. If you are observing a queue size across a group of machines and you are interested in knowing the distribution of queue sizes across those machines, use ValueObserver.

Commented for tracking/discussion in a spec issue https://github.com/open-telemetry/opentelemetry-specification/issues/818#issuecomment-681149833

Reply:
Ok, changed them. PTAL and let me know it this is what you are expecting :+1: